docs/reference/python/v0/secrets/secret.md

```diff
@@ -0,0 +1,59 @@
+---
+title: secret()
+description: Creates a reference to a secret in the secrets manager.
+---
+
+Creates a reference to a secret in the secrets manager.
+
+```python
+from nitric.resources import secret
+
+latest = secret("database.password").allow('accessing')
+```
+
+## Parameters
+
+---
+
+**name** required `string`
+
+The unique name of this secret within the secrets manager.
+
+---
+
+## Access
+
+All Nitric resources provide access permissions you can use to specify the level of access your code needs to the resource. See here for details [Access Control documentation](../../../../access-control).
+
+### Available permissions:
+
+---
+
+**put**
```

Comment:
```suggestion
**putting**
```

Reply:
Reflective of the method name `secretVersion.put()`